tests/driver_enc28j60/commands.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2022 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for netdev_eth test application
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "net/ethernet/hdr.h"
+#include "net/ethertype.h"
+#include "net/l2util.h"
+#include "byteorder.h"
+#include "shell.h"
+#include "iolist.h"
+#include "assert.h"
+
+#include "common.h"
+
+static int cmd_print_addr(int argc, char **argv);
+static int cmd_txtsnd(int argc, char **argv);
+
+const shell_command_t shell_commands[] = {
+    { "send", "Send an Ethernet frame", cmd_txtsnd },
+    { "address", "Print Ethernet address", cmd_print_addr },
+    { NULL, NULL, NULL }
+};
+
+static int cmd_print_addr(int argc, char **argv)
+{
+    (void) argc;
+    (void) argv;
```

Comment:
```suggestion
    (void)argv;
```

A cast should IMO be styled like a unary operator. And we have no space after a unary operator, so that one can immediately see that `*ptr` is dereferencing `ptr` while `a * b` is multiplication, despite both operators use the same char.